internal/namespaces/help/custom.go

```diff
@@ -0,0 +1,20 @@
+package help
+
+import (
+	"github.com/scaleway/scaleway-cli/internal/core"
+)
+
+func GetCommands() *core.Commands {
+	return core.NewCommands(
+		helpRoot(),
+		helpOutput(),
```

Comment:
Can we have something simpler for this package so we only focus on the doc here?

e.g.:
```Go
	return core.NewCommands(
		helpRoot(),
		newHelpCommand("output", outputShort, outputLong),
		newHelpCommand("cache", cacheShort, cacheLong),
		...
	)
```